src/Uno.UI/UI/Xaml/Media/Animation/ColorKeyFrame.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Uno;
+using Uno.Extensions;
+using NotImplementedException = System.NotImplementedException;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	public abstract partial class ColorKeyFrame : DependencyObject
+	{
+		public ColorKeyFrame()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of this constructor to 'protected'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)